src/validators/javascript.js

```diff
@@ -20,7 +20,7 @@ export default class JavaScriptScanner {
       // pass it the entire source file as a string.
       let eslint = new ESLint.CLIEngine({
         ignore: false,
-        rulePaths: [path.join('src', 'rules')],
+        rulePaths: [path.join('src', 'rules', 'javascript')],
```

Comment:
I always worry about Windows users, in case a module (like `ESLint.CLIEngine`) isn't normalising the paths itself.